src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -83,7 +94,7 @@ private synchronized void startVideo(final int deviceNumber) throws IOException
         try {
             grabber.start();
         } catch (FrameGrabber.Exception e) {
-            throw new IOException("A problem occurred trying to start the frame grabber for device number " + deviceNumber, e);
+            throw new IOException("A problem occurred trying to start the frame grabber", e);
```

Comment:
This should indicate what device has caused the issue. Either the URL or the device number?